playwright/setup/global-setup.ts

```
@@ -41,6 +41,35 @@ async function globalSetup(config: FullConfig) {
+      localStorage.setItem('chrome:disable-error-overlay', 'true');
+
+      // Enable allservices redesign flag for testing
+      // Unleash stores toggles in a repository object
+      const unleashKey = 'unleash:repository:repo';
+      const existingData = localStorage.getItem(unleashKey);
+      let unleashData = existingData ? JSON.parse(existingData) : { toggles: [] };
+
+      // Add or update the redesign flag
+      const redesignFlagIndex = unleashData.toggles?.findIndex(
+        (t: unknown) => (t as { name?: string }).name === 'platform.chrome.allservices.redesign'
+      );
```

ESLint

[error] Replace `⏎········(t:·unknown)·=>·(t·as·{·name?:·string·}).name·===·'platform.chrome.allservices.redesign'⏎······` with `(t:·unknown)·=>·(t·as·{·name?:·string·}).name·===·'platform.chrome.allservices.redesign'`

(prettier/prettier)